guides/hacking.md

```diff
@@ -32,21 +32,21 @@ When writing SCSS, it's useful to run both commands in separate terminals so cha
 
 ### Via Docker
 
-It can also be run on other operating systems by first [installing Docker](https://docs.docker.com/engine/installation/) (Linux users may need to [add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)), and then using the `./run` script:
+It can also be run on other operating systems by first [installing Docker](https://docs.docker.com/engine/installation/) (Linux users may need to [add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)) and then [installing dotrun](https://github.com/canonical/dotrun#installation):
```

Comment:
The section above is `Via dotrun` and is currently practically a duplicate. Running via dotrun requires docker.
So I think they should be merged.

Also the section above mentions "dotrun snap" - it's not a snap anymore, so we should not call it that.

Reply:
Yeah good idea! I've merged them.